src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -51,38 +60,33 @@ public function getBlockPrefix()
     }
 
     /**
-     * Returns the timezone choices.
-     *
-     * The choices are generated from the ICU function
-     * \DateTimeZone::listIdentifiers(). They are cached during a single request,
-     * so multiple timezone fields on the same page don't lead to unnecessary
-     * overhead.
+     * Returns a normalized array of timezone choices.
      *
      * @return array The timezone choices
+     *
+     * @internal This method is public to be used as callback and is not meant for user-land code.
      */
-    private static function getTimezones()
+    public static function getTimezones()
```

Comment:
couldn't we use a closure accessing to the private method to keep this private ?